base/binaryplatforms.jl

```diff
@@ -186,7 +186,7 @@ end
 function Base.show(io::IO, p::Platform)
     str = string(platform_name(p), " ", arch(p))
     # Add on all the other tags not covered by os/arch:
-    other_tags = sort(collect(filter(kv -> kv[1] ∉ ("os", "arch"), tags(p))))
+    other_tags = sort!(collect(filter(kv -> kv[1] ∉ ("os", "arch"), tags(p))))
```

Comment:
```suggestion
    other_tags = sort(filter(kv -> kv[1] ∉ ("os", "arch"), tags(p))))
```
You can sort iterables! #46104

Reply:
Oh nice, thanks!